shared/mail/inbucket.go

```diff
@@ -74,11 +74,8 @@ func GetMailBox(email string) (results JSONMessageHeaderInbucket, err error) {
 
 	var record JSONMessageHeaderInbucket
 	err = json.NewDecoder(resp.Body).Decode(&record)
-	switch {
-	case err == io.EOF:
```

Comment:
lol this was strange

Reply:
I thought it was maybe some left over code from when it was handled differently?

Reply:
yeah possibly